### ISSUE TITLE
new project, "Run" , close, reopen :  Project soandso cannot be opened

### ISSUE THREAD

**Issue by klauspdoll** [molpro/iMolpro#124]
create a new project, e.g. bla5

Hit "Run"

Quit

reopen project, error:  `Project /path/bla5.molpro cannot be opened`
Details `Cannot open project /home/linux24_i1/doll/imolpro/bla5
`

iMolpro can be closed, but the process is still running! Must be killed with `kill -9`

workaround: project can be opened after removing Info.plist :   

`rm /path/bla5.molpro/Info.plist`

**Comment by pjknowles** [comment]
> iMolpro can be closed, but the process is still running! Must be killed with `kill -9`

This is what is supposed to happen - the job is launched with nohup, and you can have any number of instances, including zero, of sjef watching it. Is it really true that you need `kill -9` - does the kill in iMolpro not work?

**Comment by pjknowles** [comment]
I cannot reproduce the problem at all. Maybe there is a race on writing changes to `Info.plist`. One possible remediation would be for sjef to reconstruct `Info.plist` if it finds it to be corrupt on opening.

**Comment by klauspdoll** [comment]
It runs, terminates with an error:

```
File /home/linux24_i1/doll/Molpro-latest/lib/include/bla5.xyz does not exist
 Input error

 GLOBAL ERROR Input error on processor   0                                    

```

which is correct.

But then it says on the bottom left :

`Status: run bla5_1 running`

which seems wrong.

When I quit via Project/Quit (is okay) and reopen, it gives the error `Project bla5.molpro cannot be opened`

I click "OK", iMolpro closes, but the command line still says:

`python iMolpro.py` 

I can send it to the background with `ctrl-z bg`, but then I see the process running:

```
ps -aefl|grep pyt
0 S doll      8282 14199  1  80   0 - 525783 do_sys 16:47 pts/1   00:00:00 python iMolpro.py
```

**Comment by pjknowles** [comment]
I see, no geometry file provided, though the auto-provided input refers to one. I see the same behaviour, that the status is not updated from running, even though molpro has finished. And I reproduce the inability to reopen the project.

**Comment by pjknowles** [comment]
This is a problem from sjef, caused by the recent developments to introduce exception handling that can be passed up to python. It comes because the status polling of a local job runs /bin/ps which gives an exit code of 1, which previously was ignored, but now throws an exception. Logged as https://github.com/molpro/sjef/issues/273

**Comment by pjknowles** [comment]
Should be fixed with sjef 1.37.5

**Comment by pjknowles** [comment]
I made binaries: https://github.com/molpro/iMolpro/releases/tag/0.3.26 so you can try. I think it's fixed, but please reopen if not.
